flutter/lib/src/default_integrations.dart

```diff
@@ -76,6 +81,14 @@ class FlutterErrorIntegration extends Integration<SentryFlutterOptions> {
 
     options.sdk.addIntegration('flutterErrorIntegration');
   }
+
+  @override
+  void close() {
+    /// Restore default
+    FlutterError.onError = defaultOnError;
+    defaultOnError = null;
+    super.close();
```

Comment:
this line does nothing I guess, hence not necessary

Reply:
Makes sense to forward the call to the base when you're overriding. Think like: At this location you're unaware of what's `base` and it also can change int the future

Reply:
it's an interface though, fine with it, but it's the only integration doing it so far.